engine/paymail/paymail_service_client.go

```diff
@@ -184,3 +217,14 @@ func (s *service) extractP2P(capabilities *paymail.CapabilitiesPayload) (success
 	}
 	return
 }
+
+func (s *service) validatePaymentDestinationResponse(response *paymail.PaymentDestinationResponse, satoshis uint) error {
+	var sum uint64
+	for _, out := range response.Outputs {
+		sum += out.Satoshis
+	}
+	if sum != uint64(satoshis) {
```

Comment:
I think, if we design it better - the conversion will not be necessary.

Reply:
I'm using now bsv.Satoshis from models package